cmd/object-handlers.go

```diff
@@ -820,6 +970,11 @@ func (api objectAPIHandlers) PutObjectHandler(w http.ResponseWriter, r *http.Req
 		return
 	}
 
+	if objInfo.IsCompressed() {
+		// Ignore compressed ETag.
+		objInfo.ETag = "-" + objInfo.ETag
```

Comment:
ETag to be ignored is of the form `ETag-1` not `-ETag`

Reply:
The gateway (Azure) parts had `ETag-1`. So changing it will disturb the azure flow

Reply:
`-ETag` was also working ! 
changed anyways .. Done!